graphql_server/aiohttp/graphqlview.py

```diff
@@ -162,7 +159,21 @@ async def __call__(self, request):
             )
 
             if is_graphiql:
-                return await self.render_graphiql(params=all_params[0], result=result)
+                graphiql_data = GraphiQLData(  # type: ignore
+                    result=result,
+                    subscription_url=self.subscriptions,
+                    **all_params[0]._asdict()  # noqa
```

Comment:
Since `GraphiQLData` is a TypedDict with a finite number of fields I think it would be better to pass each of the fields explicitly rather than using `**`